channel/adjudicator.go

```diff
@@ -36,30 +36,52 @@ type (
 	// AdjudicatorEvents. Those events might be triggered by a Register or
 	// Progress call on the adjudicator from any channel participant.
 	Adjudicator interface {
-		// Register should register the given ledger channel state on-chain.
-		// If the channel has locked funds into sub-channels, the corresponding
-		// signed sub-channel states must be provided.
+		Registerer
+		Withdrawer
+		Progresser
+		AdjudicatorEventSubscriber
+	}
+
+	// RegistererSubscriber is used the sub-set of Adjudicator interface
+	// required for watching the blockchain for adjudicator events and
+	// registering disputes if required.
+	RegistererSubscriber interface {
```

Comment:
Is this used anywhere yet? If not, introduce with the change that will use it?

Reply:
Yes, this is used in #182. I think, we `RegistererSubscriber` interface alone to that PR ?

Reply:
ok